src/Symfony/Bridge/PhpUnit/bootstrap.php

```diff
@@ -4,7 +4,7 @@
 use Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
 
 // Detect if we're loaded by an actual run of phpunit
-if (!class_exists('PHPUnit_TextUI_Command', false)) {
+if (!defined('PHPUNIT_COMPOSER_INSTALL') && !class_exists('PHPUnit_TextUI_Command', false)) {
```

Comment:
Where is this constant defined and is it guaranteed that it won't be removed in the future?